### PR TITLE
Integration data entry

### PR DESCRIPTION
**Pull request by hclarkEnq**

### CODE REVIEW

**Review by adamloup-enquizit — DISMISSED** [review]
👍

File: apps/modernization-ui/src/pages/addPatient/components/Race/RaceFields.tsx [hunk 1]
```diff
@@ -2,30 +2,78 @@ import { Checkbox, Fieldset, Grid } from '@trussworks/react-uswds';
...
+                                        <SearchCriteriaContext.Consumer>
+                                            {({ searchCriteria }) =>
+                                                Object.values(searchCriteria.races).map((race, index) => (
+                                                    <Checkbox
+                                                        key={index}
+                                                        onChange={(e) => {
+                                                            if (!value || value?.length === 0) {
+                                                                if (e.target.checked) {
+                                                                    tempArr.push(e.target.value);
+                                                                    console.log('true:', tempArr);
```
Comment (stevegsa) [inline apps/modernization-ui/src/pages/addPatient/components/Race/RaceFields.tsx:34]: do we want to keep these log statements in?

**Review by stevegsa — APPROVED** [review]
Looks great!!

File: apps/modernization-ui/src/pages/addPatient/components/Race/RaceFields.tsx (hunk 1 above)
Comment (hclarkEnq) [reply apps/modernization-ui/src/pages/addPatient/components/Race/RaceFields.tsx:34]: Not needed, will remove

**Review by JimBarrows-Enquizit — DISMISSED** [review]

**Review by mpeels — APPROVED** [review]

**Pull request merged**